README.md

```diff
@@ -315,3 +315,56 @@ docker run -d --restart always --name github-runner \
   -v /tmp/github-runner-your-repo:/tmp/github-runner-your-repo \
   myoung34/github-runner:latest
 ```
+
+## Ephemeral mode
+
+GitHub's hosted runners are completely ephemeral.  You can `sudo rm -rf /` without breaking all future jobs.
```

Comment:
```suggestion
GitHub's hosted runners are completely ephemeral.  You can remove all its data without breaking all future jobs.
```

I wouldnt want someone ever thinking `rm -rf /` is a good idea even in a container 😆